R/common-column-row-expectations.R

```diff
@@ -95,4 +95,29 @@ returnColsWithMoreThanFiftyCategories <- function(df) {
     }
   }
   colList
-}
\ No newline at end of file
+}
+
+#' @title
+#' Remove columns with DTS suffix
+#' @description Remove columns with DTS in the suffix of the column name
+#' @param df A data frame to be altered
+#' @return The input data frame with DTS suffix columns removed
+#'
+#' @export
```

Comment:
Can you remove this line? It will prevent the function from showing up in the namespace.

Reply:
I removed all three of these lines. Is that what you meant? Or did you just mean the export line?